dora/active_sampling/lin_sampler.py

```diff
@@ -0,0 +1,391 @@
+""" Bayesian Linear Sampler Module. """
+import logging
+
+import numpy as np
+import scipy
+import scipy.stats as stats
+import scipy.linalg as linalg
+
+from dora.active_sampling.base_sampler import Sampler, random_sample
+
+log = logging.getLogger(__name__)
+
+class BayesianLinear(Sampler):
+    """BayesianLinear Class.
```

Comment:
"Bayesian linear model" would probably be a more appropriate description of the class.